src/models/safe.ts

```diff
@@ -0,0 +1,31 @@
+export type GnosisSafe = {
+  address: string
+  nonce: number
+  owners: string[]
+  threshold: number
+}
+
+export interface SafeTransactionType {
+  nonce: number
+  origin: string
+  data?: string
+  dataDecoded?: {
+    method: string
+    parameters: object[]
+  }
+  isExecuted: boolean
+  safeTxGas: number
+  safeTxHash: string
+  submissionDate: string
+  executionDate: string
+  confirmations?: {
+    owner: string
+    submissionDate: string
+    transactionHash: string
+    signature: string
+    signatureType: string
+  }[]
+  threshold?: number
+  isHistory?: boolean // when nonce is less than current safe nonce
```

Comment:
is this our data or data that comes from the Safe API?

I would keep this type as close to the Safe API response is possible. Maybe we can have another prop to whatever component needs it: `isPastTransaction` or something.